### PR TITLE
attestation/pytdxmeasure: parse module version

### PR DESCRIPTION
**Pull request by HaokunX-intel**
parse and return the valid ModuleVersion

### CODE REVIEW

File: attestation/pytdxmeasure/pytdxmeasure/utility.py [hunk 1]
```diff
@@ -207,3 +208,62 @@ def get_tee_tcb_info_valid_val(self):
         elif self.device_node_name == DEVICE_NODE_NAME_1_5:
             val = TCB_INFO_VALID_VAL_1_5
         return val
+
+class ModuleVersion():
```
Comment (kenplusplus) [inline attestation/pytdxmeasure/pytdxmeasure/utility.py:212]: do not need ()

File: attestation/pytdxmeasure/pytdxmeasure/tdreport.py [hunk 2]
```diff
@@ -120,6 +124,18 @@ def parse(self):
             self.tee_tcb_svn2, offset = self.get_bytes(offset, 0x10)
             self.reserved, offset = self.get_bytes(offset, 0x5f)
 
+        # parse module svn
+        self.module_version, _ = ModuleVersion.try_get_module_version(self.tee_tcb_svn)
+
+    def try_get_module_version(self):
```
Comment (kenplusplus) [inline attestation/pytdxmeasure/pytdxmeasure/tdreport.py:130]: I dislike this function, look wierd design and confuse with below ModuleVersion class.

**Review by kenplusplus — CHANGES_REQUESTED** [review]

File: attestation/pytdxmeasure/pytdxmeasure/utility.py (hunk 1 above)
Comment (HaokunX-intel) [reply attestation/pytdxmeasure/pytdxmeasure/utility.py:212]: Sure.

File: attestation/pytdxmeasure/pytdxmeasure/tdreport.py (hunk 2 above)
Comment (HaokunX-intel) [reply attestation/pytdxmeasure/pytdxmeasure/tdreport.py:130]: Sure. It looks a little confusing to share the same function name. The function here is redundant, can be erased.

**Review by kenplusplus — APPROVED** [review]
LGTM

**Pull request merged**